dags/utils/gcp.py

```diff
@@ -91,6 +89,9 @@ def export_to_parquet(
 
     params = get_dataproc_parameters("google_cloud_airflow_dataproc")
 
+    if arguments is None:
+        arguments = []
+
```

Comment:
We'll need to do the same for `static_partitions`, which is concatenated to a list on line 142